client/my-sites/checkout/src/components/checkout-terms.tsx

```diff
@@ -59,6 +63,53 @@ export default function CheckoutTerms( { cart }: { cart: ResponseCart } ) {
 
 	const showToSFoldableCard = useToSFoldableCard();
 
+	const shouldShowRefundPolicy =
+		! isJetpackCheckout() && ! isJetpackNotAtomic && ! isAkismetCheckout();
+	const shouldShowInternationalFeeNotice =
+		! isJetpackCheckout() && ! isJetpackNotAtomic && ! isAkismetCheckout();
+
+	const shouldShowBundledDomainNotice = () => {
```

Comment:
I assume you want this to be an IIFE? If so, you need to wrap the function in parens and call it. Alternatively, you can call it when you need it below.